src/Components/OATPropertyEditor/FormUpdateProperty.tsx

```diff
@@ -220,16 +231,25 @@ export const FormUpdateProperty = ({
                         : multiLanguageSelectionsDisplayName,
                 writable,
                 unit: activeNestedProperty.unit,
-                '@id': id ? id : activeNestedProperty['@id'],
                 schema: activeNestedProperty.schema,
                 name: activeNestedProperty.name
             };
 
+            // Update nested enum value
+            if (
+                activeNestedProperty &&
+                activeProperty.schema['@type'] === 'Enum'
+            ) {
+                delete prop.displayName;
```

Comment:
Why is display name being removed? It's a valid DTDL prop right?

Reply:
It is a valid prop, it's an optional one. From the card, I understood we would take it off. 
Let me know if I should re add it for the update modal? @jamescarpinter @jamiewilbraham